docs/samples-and-tutorials/index.md

```diff
@@ -20,9 +20,9 @@ This guide shows you how to create an ASP.NET Core web app and associated unit t
 
 ### Tutorials
 
-**[Writing .NET Core console apps using the .NET Core CLI: A step-by-step guide](../core/tutorials/cli-create-console-app.md)**
+**[Tutorial: Create a .NET Core console application using Visual Studio Code](../core/tutorials/with-visual-studio-code.md)**
 
-This guide shows you how to use the .NET Core CLI tooling to build cross-platform console apps. It starts with a basic console app and eventually spans multiple projects, including testing. You add features step-by-step, building your knowledge as you go. The [completed sample](https://github.com/dotnet/samples/tree/master/core/console-apps) is available in the dotnet/samples repository on GitHub.
+This tutorial shows how to create and run a .NET Core console application by using Visual Studio Code and the .NET Core CLI. Project tasks, such as creating, compiling, and running a project are done by using the .NET Core CLI.
 
 **[Writing Libraries with Cross Platform Tools](../core/tutorials/libraries.md)**
```

Comment:
If you update the description of this one too like the above, this PR will fix https://github.com/dotnet/docs/issues/20185. Or you'd like to address that separately?

Reply:
We can do that in a separate PR.